concepts.wip/if-expressions/about.md

```diff
@@ -1,52 +1,87 @@
 # About
 
-`if`-expressions in Julia are similar to those seen in other languages:
+In Julia, like many other languages, allow for the following with `if`-expressions:
+
+- Chained if-expressions
+- Nested if-expressions
+
+~~~~exercism/note
+[`if`-expressions](https://docs.julialang.org/en/v1/manual/control-flow/#man-conditional-evaluation) as defined here will encompass the following syntaxes:`if`, `if-else` and `if-elseif-else` statements.
+~~~~
+
+`if`-expressions allows for branching in our programs, given a condition is satisfied.
+
+## A first example
+
+The `function` defined below is not very realistic, but it shows the anatomy of an `if` statement. 
+
+```julia
+julia> function password_checker(password)
+           if password == system_password
+               return "Welcome back!"
+            end
+        end
+
+# Assume system_password is "abc123"
+julia> password_checker("abc123")
+Welcome back!
+
+julia> password_checker("xyz321")
+
+```
+`println("Welcome back!")` is run, if the `password == system_password`, otherwise it is skipped, and nothing is printed to the screen.
```

Comment:
```suggestion
`"Welcome back!"` is returned if `password == system_password`, otherwise that block is skipped and `nothing` is returned.
```

Func returns a string now. Maybe leave out the bit about `nothing` being returned.